### PR TITLE
foo2zjs: 20180519 -> 20210116

### PR DESCRIPTION
**Pull request by alyssais**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 118274` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>1 package failed to build and are new build failure:</summary>
  <ul>
<li>foo2zjs: <a href="https://termbin.com/t48s">plain log</a>  |  <a href="https://htmlpreview.github.io/?https://gist.githubusercontent.com/SuperSandroBot/71f4ae4a0f2c5148bfb550d42787b83e/raw/index.html">pretty log </a></li>
  </ul>
</details>

File: pkgs/misc/drivers/foo2zjs/default.nix [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
+    url = "http://www.loegria.net/mirrors/foo2zjs/foo2zjs-${version}.tar.gz";
+    sha256 = "14x3wizvncdy0xgvmcx541qanwb7bg76abygqy17bxycn1zh5r1x";
   };
 
   nativeBuildInputs = [ unzip ];
   buildInputs = [ foomatic-filters bc ghostscript systemd vim ];
 
   patches = [ ./no-hardcode-fw.diff ];
 
+  enableParallelBuilding = true;
```
Comment (7c6f434c) [inline pkgs/misc/drivers/foo2zjs/default.nix:17]: Hmm, it looks like it mostly works for `-j2` and `-j3`, but not above

**Comment by 7c6f434c** [comment]
@SuperSandro2000 thanks for doing the checks with high parallelism enabled!

**Comment by SuperSandro2000** [comment]
> @SuperSandro2000 thanks for doing the checks with high parallelism enabled!

Your welcome. I think it runs with 8 cores but I am not fully sure tbh.

**Pull request merged**